project/Settings.scala

```diff
@@ -123,7 +123,8 @@ object Settings {
 
     scalacOptions ++= Seq(
       "-Xsource:2.12",
-      "-Xmax-classfile-name", "240"
+      "-Xmax-classfile-name", "240",
+      "-target:jvm-1.8"
```

Comment:
I've been looking a bit around this, and it might cause conflicts when we trying to support Scala 2.13. I'd suggest we remove this setting and do the following when declaring the Map in the client:

```scala
val params = Map("fields" -> fields.iterator.map(_.value).toList.asJava).asJava
```

:)

Reply:
Good idea :+1: